docassemble/MotionToDismissCRA/data/questions/CRA_Motion_to_Dismiss.yml

```diff
@@ -35,38 +35,35 @@ metadata:
   short title: |
     Dismiss CRA
   description: |
-    If you have a Child Requiring Assistance (CRA) case, you can use this form to ask the court 
-    to dismiss it. You can use this form if you are the child, the child's parent or legal guardian, 
-    the child's custodian, or the person who asked for the CRA.
+    If the court has created a Child Requiring Assistance (CRA) case against a child, you can use this form to ask the court to dismiss it.
   can_I_use_this_form: |
     You can use this interview if you are:
 
-    * The child,
-    * The child's parent or legal guardian,
-    * The child's custodian, or
-    * The person who asked for the CRA.
+    * the child,
+    * the child's parent or legal guardian,
+    * the child's custodian, or
+    * the person who asked for the CRA.
   before_you_start: |
-    Before you start, you need to know the date that the application for a CRA case was filed and 
-    the case number for the CRA case.
+    Before you start, you need to know:
+    * the date that the application for a CRA case was filed, and 
+    * the case number for the CRA case. This is also called a docket number.
   maturity: production
   estimated_completion_minutes: 15
   estimated_completion_delta: 3
   languages:
     - en
-  help_page_url: "https://www.mass.gov/info-details/learn-about-your-rights-in-a-child-requiring-assistance-case"
-  help_page_title: "Learn about your rights in a Child Requiring Assistance Case"
+  help_page_url: https://www.mass.gov/info-details/learn-about-your-rights-in-a-child-requiring-assistance-case
+  help_page_title: Learn about your rights in a Child Requiring Assistance Case
   LIST_topics: 
-    - FA-00-00-00-00
-    - CR-10-00-00-00
+    - FA-07-00-00-00
```

Comment:
I think the previous tags might have worked better for this interview. (Don't forget to change the plain `tags` fallback property, as well.)

```
- FA-00-00-00-00
- CR-10-00-00-00
```